src/OSPSuite.InstallationValidator.Core/Services/BatchStarterTask.cs

```diff
@@ -16,11 +16,13 @@ public class BatchStarterTask : IBatchStarterTask
    {
       private readonly IInstallationValidationConfiguration _applicationConfiguration;
       private readonly IStartableProcessFactory _startableProcessFactory;
+      private readonly ILogWatcherFactory _logWatcherFactory;
 
-      public BatchStarterTask(IInstallationValidationConfiguration applicationConfiguration, IStartableProcessFactory startableProcessFactory)
+      public BatchStarterTask(IInstallationValidationConfiguration applicationConfiguration, IStartableProcessFactory startableProcessFactory, ILogWatcherFactory logWatcherFactory)
       {
          _applicationConfiguration = applicationConfiguration;
          _startableProcessFactory = startableProcessFactory;
+         _logWatcherFactory = logWatcherFactory;
       }
 
       private string batchToolPath => Path.Combine(baseInstallPath, Constants.Tools.PKSimBatchTool);
```

Comment:
that baseInstallPath is actually PKSim InstallPath correct? If yes, please rename because it feels like this is where the InsallationValidator tool is installed. (they won;'t be installed in the same directory)